manifests/config.pp

```diff
@@ -0,0 +1,36 @@
+# @private
+#
+# This class is used to configure the jmxtrans config files
+#
+class jmxtrans::config {
+  include ::jmxtrans
+
+  if $::jmxtrans::manage_service_file {
+    case $::service_provider {
```

Comment:
Elsewhere, we use the `$facts['service_provider']` syntax, so for consistency, we should probably use that here.